### PR TITLE
Update URL.origin description and example usage

### PR DESCRIPTION
**Pull request by lem-onade**
### Description

The current example suggests the port is removed when using `URL.origin`, which is not always the case. Only when the scheme (http, https) matches its default port (80, 443), is the port removed.

The current structure description suggests default ports (http - 80, https - 443) would not be removed, however they are. 

For other scheme/port combinations, the port is not removed.

### Motivation

Additional example usage could help with understanding how `URL.origin` behaves in different circumstances.

The current structure description did not match actual behaviour, which could be confusing.

### Additional details

Example usage: https://github.com/mdn/content/pull/24792#issuecomment-1441759015

### Related issues and pull requests

/

### CODE REVIEW

**Comment by teoli2003** [comment]
It does:

Capture d’écran 2023-02-23 à 14 04 30

**Comment by lem-onade** [comment]
> It does:
> 
> Capture d’écran 2023-02-23 à 14 04 30

My bad, it does for https + :443 and http + :80

image

However, for other combinations, it seems to not cut off the port.

image

Which seems to go against the structure description:

> The exact structure varies depending on the type of URL:
>
> For http or https URLs, the scheme followed by '://', followed by the domain, followed by ':', followed by the port (the default port, 80 and 443 respectively, if explicitly specified).
> ...

This suggests :80 and :443 would not be removed if explicitly specified. Perhaps I am misinterpreting this?

**Comment by teoli2003** [comment]
Yes, we should fix the sentence.

**Comment by lem-onade** [comment]
I've added a suggestion for the incorrect sentence, as well as some more example usage!

File: files/en-us/web/api/url/origin/index.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ varies depending on the type of URL:
 
 - For `http` or `https` URLs, the scheme followed by
   `'://'`, followed by the domain, followed by `':'`, followed by
-  the port (the default port, `80` and `443` respectively, if
-  explicitly specified).
+  the port (if explicitly specified, unless it is the default port - `80` and `443` respectively).
+-
```
Comment (teoli2003) [inline files/en-us/web/api/url/origin/index.md:20]:
```suggestion
```

File: files/en-us/web/api/url/origin/index.md [hunk 2]
```diff
@@ -34,6 +33,12 @@ A string.
 ```js
 const url = new URL("blob:https://mozilla.org:443/")
 console.log(url.origin); // Logs 'https://mozilla.org'
+
+const url = new URL("http://localhost:80/")
+console.log(url.origin); // Logs 'http:/localhost'
```
Comment (teoli2003) [inline files/en-us/web/api/url/origin/index.md:38]:
```suggestion
console.log(url.origin); // Logs 'http://localhost'
```

**Review by teoli2003 — APPROVED** [review]
I fixed 2 nits (not important things) and let's wait for the test to run before merging this!

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! 🎉 

(If you are looking to contribute further, here is a list of possible tasks: https://github.com/orgs/mdn/projects/25 )

**Pull request merged**